docs/src/appendix/cast/script.md

```diff
@@ -1,7 +0,0 @@
-# `script`
```

Comment:
This is temporary, until we have `run` command, right? I guess we could leave this, have it to act like a kind of overview (like for example in `account`), link to init + store `script` in here as well. wdyt?